src/Symfony/Component/Security/Csrf/composer.json

```diff
@@ -17,8 +17,8 @@
     ],
     "require": {
         "php": ">=5.3.9",
-        "symfony/security-core": "~2.4|~3.0.0",
-        "paragonie/random_compat": "~1.0"
```

Comment:
Is removing random_compat from the CSRF module intentional? In most situations this should already be available through other components.